runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -24,30 +24,38 @@
  */
 package net.runelite.client.plugins.tearsofguthix;
 
-import java.time.Instant;
-import java.util.HashMap;
-import java.util.Map;
-import javax.inject.Inject;
+import com.google.inject.Provides;
 import lombok.Getter;
-import net.runelite.api.Client;
-import net.runelite.api.DecorativeObject;
-import net.runelite.api.ObjectID;
+import net.runelite.api.*;

```

Comment:
Don't star-import non-static imports. [See our code conventions for more info.](https://github.com/runelite/runelite/wiki/Code-Conventions)